models/question.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const Option = require('./option');
+
+const QuestionSchema = new mongoose.Schema({
+    _courseId : {
+        type: mongoose.SchemaTypes.ObjectId,
+        required: true
+    },
+    question: {
+        type: String,
+        required : true
+    },
+    options : [{
+        type : mongoose.SchemaTypes.ObjectId,
+        ref : 'Option'
+    }],
+    answer : {
```

Comment:
No point storing (correct)answer here since we have it already in the options schema.